client/src/components/GraphViewerComponent/GraphViewerCytoscapeComponent/GraphViewerCytoscapeComponent.js

```diff
@@ -419,6 +419,10 @@ export class GraphViewerCytoscapeComponent extends React.Component {
     return (colors[(colors.length - 1) - im]);
   }
 
+  getModelColor(modelId) {
```

Comment:
would it be more efficient to return all the colors and then access them locally in code snippet below?

![image](https://user-images.githubusercontent.com/13650672/149260638-ad8e2638-339c-4eab-bc53-30b3bbd96ba0.png)

seems a bit heavy keep going to the settings service each time?

Reply:
Updated to get the full list before